tools/deploy-with-olm.sh

```diff
@@ -79,9 +85,10 @@ blue "Creating catalogsource containing needed dependencies on cluster"
 exit_on_err ${K8S_CLIENT} -n ${TARGET_NAMESPACE} apply -f ./tools/yamls/ocs-catalogsource.yaml
 
 blue "Creating secrets needed for the operator to run without error"
-exit_on_err ${K8S_CLIENT} -n ${TARGET_NAMESPACE} create secret generic addon-${ADDON_NAME}-parameters --from-literal=size=${CLUSTER_SIZE} --dry-run=client -o yaml | ${K8S_CLIENT} -n ${TARGET_NAMESPACE} apply -f -
+exit_on_err ${K8S_CLIENT} -n ${TARGET_NAMESPACE} create secret generic addon-${ADDON_NAME}-parameters --from-literal=size=${CLUSTER_SIZE} --from-literal=notification-email-0=${ALERT_EMAIL} --dry-run=client -o yaml | ${K8S_CLIENT} -n ${TARGET_NAMESPACE} apply -f -
```

Comment:
Where does the ALERT_EMAIL value come from? I couldn't find a definition for it. 
Are we assuming empty string if the person running the script does not provide an email in an env var?

Reply:
Updated ALERT_EMAIL to have a default value